modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -20,45 +20,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 .Procedure
 
-. Create the `Secret` CR and the `BareMetalHost` CR.
-
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
++
+.Static configuration `bmh.yaml`
 +
 [source,yaml]
 ----
 ---
```

Comment:
Is this defintion separator (---) needed? Does the file not start with `apiVersion`?